src/com/google/javascript/jscomp/parsing/IRFactory.java

```diff
@@ -143,15 +143,6 @@
       "Non-JSDoc comment has annotations. " +
```

Comment:
https://github.com/google/closure-compiler/blob/master/src/com/google/javascript/jscomp/RhinoErrorReporter.java#L116

In RhinoErrorReporter, we turn some parser warnings into diagnostic-types by regex matching the warning text :). Now that you've moved some of these to checkjsdoc, you can remove the relevant parts of the code from rhinoerrorreporter.